services/shared/src/main/java/com/wedriveu/services/shared/entity/VehicleCounter.java

```diff
@@ -0,0 +1,83 @@
+package com.wedriveu.services.shared.entity;
+
+/**
+ * @author Stefano Bernagozzi
+ */
+
+public class VehicleCounter {
```

Comment:
Please, at least implement the `toString` method, and also the `equals` if you can as it should be a standard for custom Java classes. (together with `hashcode`)